### ISSUE TITLE
Compatibility Gerrit and oauth provider

### ISSUE THREAD

**Issue by karooga** [davido/gerrit-oauth-provider#106]
It looks like the compatibility of the plugin is dependent on the version of Gerrit used?

So the latest plugin version is 2.14.6.2:

https://github.com/davido/gerrit-oauth-provider/releases/tag/v2.14.6.2

and Gerrit 2.14.6:

https://www.gerritcodereview.com/releases/2.14.md

Despite Gerrit 2.14.7 being available as well as Gerrit 2.15.1:

https://www.gerritcodereview.com/releases/2.15.md

**Comment by mwebber** [comment]
I can confirm that the latest release of this plugin (2.14.6.2) works well with both Gerrit 2.14.7 and 2.15 (haven't tested 2.15.1).
The OAuth Provider plugin is only fairly weakly dependent on the Gerrit version, so a new release of OAuth Provider is not usually required just because Gerrit has changed.

I agreed that we could make this clearer here.

**Comment by karooga** [comment]
That's good to know.  Thanks @mwebber.

**Comment by davido** [comment]
Good point. I will release new plugin versions built against the latest API versions.

As @mwebber pointed out, it is safe to use the latest released plugin version. Another option would be to fetch the binary artefact from GerritForge CI: [1], from master (as you can see, there is no specific stable-2.15 branch and version on the CI).

* [1] https://gerrit-ci.gerritforge.com/view/Plugins-master/job/plugin-oauth-bazel-master-master

**Comment by zesstra** [comment]
Thanks for this info as well, very useful. :-) +1 for including it in the documentation.
Now I can confirm that 2.14.6.2 works (as expected by you) for gerrit 2.15.1.
